README.md

```diff
@@ -1,27 +1,35 @@
 <img src="./docs/img/openshift.png" width="180" align="right">
 
-# Odo - Developer Focused CLI for OpenShift
+# Odo - Developer focused CLI for OpenShift
 
 [![Build Status](https://travis-ci.org/redhat-developer/odo.svg?branch=master)](https://travis-ci.org/redhat-developer/odo) [![codecov](https://codecov.io/gh/redhat-developer/odo/branch/master/graph/badge.svg)](https://codecov.io/gh/redhat-developer/odo) [![CircleCI](https://circleci.com/gh/redhat-developer/odo/tree/master.svg?style=svg)](https://circleci.com/gh/redhat-developer/odo/tree/master) [![mattermost](/docs/img/mattermost.svg)](https://chat.openshift.io/developers/channels/odo)
 
-A fast iterative tool for deploying your source code straight to OpenShift.
+OpenShift Do (odo) is a  fast iterative CLI tool for developers who write, build, and deploy applications on OpenShift.
+Existing tools such as `oc` are more operations-focused and require a deep-understanding of Kubernetes and OpenShift concepts.
 
-## Features
+odo is  an opinionated CLI tool that supports fast, iterative development and that abstracts away complex Kubernetes and OpenShift concepts, thus allowing developers to focus on what's most important to them: code.
+
+odo improves the developer experience with OpenShift. It is designed to be simple and concise so that you can focus on coding rather than how to deploy your application. odo builds and deploys your code to your cluster as soon as you save your changes, giving you instant feedback to validate your changes in real-time. odo's syntax and design is centered around concepts already familiar to developers, such as: project, application and component.
+
+
+## Key features
 
 - Designed for fast, iterative development cycles
-- 100% client based. No server required within your OpenShift cluster for deployment
+- Completely client based. No server is needed within your OpenShift cluster for deployment
 - Supports multiple languages and frameworks such as Node.js, Java, Ruby, Perl, PHP and Python
-- Detect changes to your local code and deploy automatically with `odo watch`
-- List all available components and services from your OpenShift cluster
+- Detects changes to your local code and deploys automatically using `odo watch`
+- Lists all available components and services from your OpenShift cluster
 
 ## Installation
 
-> The only requirement is **OpenShift 3.9.0** and above. A recommended way of testing out and using OpenShift locally is [Minishift](https://github.com/minishift/minishift).
+.Prerequisites:
+You need **OpenShift 3.9.0** and above.
+*Note:* We recommend using [Minishift](https://github.com/minishift/minishift)to test and use OpenShift locally.
```

Comment:
Saying: We recommend using Minishift

And then saying: to test and use OpenShift locally.

Is saying the same thing.. Just saying "We recommend using Minishift" seems to convey exactly what we mean.